src/serializers/emails-serializer.js

```diff
@@ -0,0 +1,66 @@
+import { Serializer as JsonApiSerializer } from 'jsonapi-serializer';
+import _ from 'lodash';
+
+import { formatSubObjects } from 'utils/format-sub-objects';
+import { serializerOptions } from 'utils/jsonapi';
+import { openapi } from 'utils/load-openapi';
+import { apiBaseUrl, resourcePathLink, paramsLink } from 'utils/uri-builder';
+
+const emailResourceProp = openapi.components.schemas.EmailResource.properties;
+const emailResourceType = emailResourceProp.type.enum[0];
+const emailResourceAttributes = emailResourceProp.attributes;
+const emailResourceKeys = _.keys(emailResourceAttributes.properties);
+
+/**
+ * Some fields need to be prepared for the serializer
+ *
+ * @param {object[]} rawEmails raw data from data source
+ */
+const prepareRawEmails = (rawEmails) => {
+  _.forEach(rawEmails, (email) => {
+    email.preferredInd = email.preferredInd === 'Y';
+  });
```

Comment:
Seems like `lastActivityDate` is not using the format defined here: https://github.com/osu-mist/persons-api/blob/0792a3fc0212dd25f7c6f4be4df4198719d660f6/openapi.yaml#L1920